test_files/duplicated_externs.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @fileoverview A file containing externs duplicating the symbols generated from
+ * declare/declare.d.ts (and thus declare/externs.js). This is used to ensure generated externs do
+ * not conflict with pre-existing externs.
+ * @externs
+ */
+
+/** @type {number} */
+var declareGlobalVar;
```

Comment:
I vaguely recall that you need to explicitly add these files in the test suite somewhere, but I don't see a corresponding change here that does that.

Reply:
This is a bit embarrassing, I think I wanted to add this in a PR long past. I updated the test runner to include it.

Reply:
It seems there is no affected test (?).  Maybe this belongs in some other CL?